package.json

```diff
@@ -31,7 +32,8 @@
     "express": "4.21.1",
     "isbot": "4.4.0",
     "react": "18.3.1",
-    "react-dom": "18.3.1"
+    "react-dom": "18.3.1",
+    "vitest": "^2.1.6"
```

Comment:
pls pin to the exact version. I'd like us to have complete control over exact version which is why I pinned all packages to exact version

Reply:
regarding vitest vs bun. bun also has an integrated test runner and ideally, we would use that to have as little dependencies as possible. my hope is that eventually, we'd be able to get rid of node vite and even biome (if bun implements formatting and linting)

however, it is certainly possible that vitest is currently a better option. I haven't used bun test runner nor vitest enough yet so I am not sure. could you pls check and compare them a bit and then let me know what would be pros/cons of picking one/antoher and then we can decide